SGS 186

jayaajay opened the pull request:
This fixes the build failure of ibp_server with 32-bit architecture.

With Centos - 6.7 32-bit ISO image,
ibp_server fails to compile.
/usr/include/apr-1/apr.h:285:28: error: expected '=', ',', ';', 'asm' or '_attribute_' before 'apr_off_t'
This is due to the fact that not all options are passed to the compiler. 
pass the following: `/usr/local/bin/apr-1-config -cppflags -cflags` to compiler

PerilousApricot commented:
Hi @jayaajay Can you squash these commits, please? Also, we're moving away from adding extra spaces between parens and the function in cmake, could you please trim them as well? Thanks!

jayaajay commented:
Hi @PerilousApricot I have squashed my commits and fixed the extra spaces as well. If the changes are alright, could you please merge it with master, so that I can finish the packaging for 32-Bit centos. Thanks!

PerilousApricot commented:
Looks great. One last request -- when you squashed, you left the commit message to say the whole thing was the trim extra spaces. Rewrite him and we should be good to go.

jayaajay commented:
Done! Thanks for noting it.

PerilousApricot commented:
Cheers!
